### PR TITLE
Add few ways to flatten list of lists

### PR DESCRIPTION
**Pull request by lipinski**
I add a few ways to flatten list of lists.

### CODE REVIEW

**Comment by brennerm** [comment]
Please add your examples to https://github.com/brennerm/PyTricks/blob/master/flattenlist.py

**Comment by lipinski** [comment]
I moved examples to flattenlist.

File: flattenlist.py [hunk 1]
```diff
@@ -19,3 +19,22 @@ def flatten_list(L):
...
+# Solutions:
+
+[x for _list in a for x in _list]
+
+import itertools
+list(itertools.chain(*a))
+
+list(itertools.chain.from_iterable(a))
+
+reduce(lambda x, y: x+y, a)
```
Comment (brennerm) [inline flattenlist.py]: reduce is not available in Python3. Please add a comment on this.

**Comment by brennerm** [comment]
Could you please add a print() around all solutions. I want the script to generate some output.

**Comment by lipinski** [comment]
I added.

**Comment by brennerm** [comment]
@lipinski Thank you!
Want to add yourself to the contributors?

**Comment by lipinski** [comment]
It will be nice.

**Comment by brennerm** [comment]
Then please do so. Just add an entry with your name to the file CONTRIBUTORS and add the commit to this PR :)

**Comment by lipinski** [comment]
Ok. Done.

**Comment by brennerm** [comment]
:+1:

**Pull request merged**